<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>q3D2DDlg</class>
 <widget class="QDialog" name="q3D2DDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D2D</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="push_load">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Charger  les images et paramètres</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_img">
     <property name="text">
      <string>Chemin vers le dossier contenant  les images </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_param">
     <property name="text">
      <string>Chemin vers le dossier contenant les paramètres</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="push_slctPt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sélectionner un point</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="x">
       <property name="text">
        <string>x:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_x">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="y">
       <property name="text">
        <string>y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_y">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="z">
       <property name="text">
        <string>z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_z">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="push_reproj">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reprojeter</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_autoReproj">
     <property name="text">
      <string>Calcul automatique de la reprojection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_BestImg">
     <property name="text">
      <string>Affichage de la meilleure image par défaut</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Liste des images voyant le point:</string>
     </property>
     <property name="buddy">
      <cstring>label_5</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listImg"/>
   </item>
   <item>
    <widget class="QPushButton" name="push_display">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
